modules/azure/disk_test.go

```diff
@@ -0,0 +1,36 @@
+// +build azure
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package azure
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetDiskE(t *testing.T) {
```

Comment:
Is this test and the one below it just a couple unit tests checking that invalid params are handled correctly?

Reply:
Correct, without a mocking framework, we settled on negative test cases for unit test to start.